test/test_helpers/axe_helpers.rb

```diff
@@ -98,6 +98,8 @@ def click_button(locator = nil, **kwargs)
   def visit(path, **attributes)
     super
 
+    page.driver.wait_for_network_idle
```

Comment:
**praise:** This is an interesting little bit. Might prevent a lot of our issues.  Thanks for looking at these failures